readme.md

```diff
@@ -17,34 +17,33 @@
 * Define what `this` represents in the context of an event listener.
 * Describe the difference between asynchronous and synchronous program execution.
 
-## Framing (2:30 - 2:35, 5 minutes)
+## Framing (5 minutes / 0:05)
 
 In order to do things on the client side and give our web applications behavior, we need programmatic access to the HTML and CSS using Javascript. Javascript gives us not only the ability to manipulate the **DOM** as we've seen, but also to make it respond to user actions. This is where events come in: we can *listen* for certain kinds of user-driven events, such as clicking a button, entering data into a form, keypresses, and many, many more.
 
-
-## User Interaction
+### User Interaction
 
 As we write client-side Javascript (javascript that is executed by *our browsers*, as opposed to being executed by a server we are accessing), it is very important to keep the user's actions in mind when designing our app's UI.
 
-For example, let's say we have a single button on our landing page, we need to write some code that will execute whenever a user clicks on that button, i.e. pop-up an ad with a special one-and-a-lifetime promotion for that user.
+For example, let's say we have a single button on our landing page, we need to write some code that will execute whenever a user clicks on that button.
```

Comment:
@jshawl do you mean just create a page with a button that does something when the user clicks on it? or is there something specific i'm missing?